packages/framework/src/Command/MigrateUploadedFilesCommand.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Command;
+
+use League\Flysystem\FilesystemOperator;
+use League\Flysystem\UnableToDeleteFile;
+use Shopsys\FrameworkBundle\Component\Setting\Setting;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+#[AsCommand(
+    name: 'shopsys:uploaded-files:migrate',
+    description: 'Move all uploaded files from subdirectories to the top level directory.',
+)]
+class MigrateUploadedFilesCommand extends Command
```

Comment:
it would be nice to ask a developer to test this command (I can probably do that tomorrow but if I do not make it, please ask someone else as I will be off on Thursday and Friday)